### ISSUE TITLE
Support more slides types on the submit_slides page

### ISSUE THREAD

**Issue by webknjaz** [EuroPython/epcon#1063]
It is quite popular to make slides with things like [revealjs](https://revealjs.com)/[remarkjs](https://remarkjs.com)/[impressjs](https://impress.js.org).

Essentially, it's just html+js+css. But it's inconvenient to put that into a tarball for upload.

Usually, folks just use GitHub Pages to serve that.

**:bulb: Idea:** Why not allow also submitting URLs with ready-to-watch presentations? Maybe repo URLs too.

**:bulb: Better yet:** It'd be cool if there was a repository under EuroPython org where people could just contribute their presentations, maybe even hook-up some Travis CI or similar to support a build step for those.
This repository would expose them all via GitHub Pages with standardized URLs and year-based layout. Accepting PRs from slides authors could also be automated BTW: with ACL based on the PR changes directory, some robot could auto-merge stuff from speakers.
Such presentations could be exposed as https://EuroPython.github.io/talks/2019/some-talk-slug or something like https://talks.europython.eu/2019/some-talk-slug.

**Comment by umgelurgel** [comment]
Thanks for the suggestion - it's a really cool idea! The v1 is easily doable - adding a slides URL + repo url to the slides submission form. The v2 suggestion sounds a bit more involved and I'm worried about the maintenance it would require - could definitely used a hand setting it up. Is that something you think you could help with?

**Comment by webknjaz** [comment]
Yes, @artcz and a few others support the idea. I would like to help with this but right now I don't have time. If anybody will try implementing this before I have time  — I think I'd be able to at least review that/point in the right direction.

**Comment by umgelurgel** [comment]
Got it - the slide url support will be added to the ep2019 website, the autobuild could be added as a future feature if there's capacity to work on it.

**Comment by webknjaz** [comment]
@umgelurgel feel free to assign me this issue.

**Comment by webknjaz** [comment]
P.S. I'm only going to do the GH automation part. So wiring up the URL field should be done by somebody else.
